phpstan.neon

```diff
@@ -1,4 +1,7 @@
 parameters:
+    checkMissingIterableValueType: false
+    excludes_analyse:
```

Comment:
proč?

Reply:
phpcs tam chce pridat typy, a phpstan potom vypisuje ze je to nekompatibilne s nadradenou triedou. teraz to neriesim (subory zostavaju nezmenené), kedze v dalsi fazach refactoringu sa bude musiet zrusit zavislost na PDO.

Reply:
ok